README.md

```diff
@@ -11,140 +11,109 @@
 
 ## Usage
 
-`$ treefmt --help`
+In order to use `treefmt` in your project, make sure the config file `treefmt.toml` is present in the root folder and is edited to your needs. You can generate it with:
 
 ```
-treefmt 0.5.0
-✨  format all your language!
-
-USAGE:
-    treefmt [FLAGS] [OPTIONS] [--] [paths]...
-
-FLAGS:
-        --allow-missing-formatter    Do not exit with error if a configured formatter is missing
-        --clear-cache                Reset the evaluation cache. Use in case the cache is not precise enough
-        --fail-on-change             Exit with error if any changes were made. Useful for CI
-    -h, --help                       Prints help information
-        --init                       Create a new treefmt.toml
-        --no-cache                   Ignore the evaluation cache entirely. Useful for CI
-    -q, --quiet                      No output printed to stderr
-        --stdin                      Format the content passed in stdin
-    -V, --version                    Prints version information
-    -v, --verbose                    Log verbosity is based off the number of v used
-
-OPTIONS:
-        --config-file <config-file>     Run with the specified config file, which is not required to be in the tree to
-                                        be formatted
-    -f, --formatters <formatters>...    Select formatters name to apply. Defaults to all formatters
-        --tree-root <tree-root>         Set the path to the tree root directory. Defaults to the folder holding the
-                                        treefmt.toml file [env: PRJ_ROOT=]
-    -C <work-dir>                       Run as if treefmt was started in <work-dir> instead of the current working
-                                        directory [default: .]
-
-ARGS:
-    <paths>...    Paths to format. Defaults to formatting the whole tree
+$ treefmt --init
 ```
 
-## Configuration format
-
-In order to use `treefmt` in the project, `treefmt.toml` should exists in the root folder. For example, we want to use [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) on our Nix project and rustfmt on our Rust project, then the `treefmt.toml` will be written as follows:
+You can run `treefmt` in your project root folder like this:
 
 ```
-[formatter.nix]
-command = "nixpkgs-fmt"
-includes = ["*.nix"]
-
-[formatter.rust]
-command = "rustfmt"
-options = ["--edition", "2018"]
-includes = ["*.rs"]
+$ treefmt
 ```
 
-See the [wiki](https://github.com/numtide/treefmt/wiki) for more examples.
+To explore the tool’s flags and options, type:
 
-## Use cases
+```
+$ treefmt --help
+```
 
-### CLI usage
+You can also integrate `treefmt` into your CI.
```

Comment:
```suggestion
```
For now let's remove it and introduce a chapter in the docs specifically about CI support.